lldb/test/API/lang/swift/po/objc/TestSwiftPOObjC.py

```diff
@@ -0,0 +1,28 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+
+class TestSwiftPOObjC(TestBase):
+    #NO_DEBUG_INFO_TESTCASE = True
+    @skipUnlessDarwin
+    @swiftTest
+    def test(self):
+        """Test running po on a Swift object from Objective-C. This
+        should initialize a generic SwiftASTContext with no parsing of
+        Swift modules happening"""
+        self.build()
+
+        lldbutil.run_to_source_breakpoint(
+            self, "break here", lldb.SBFileSpec('main.m'))
+
+        self.expect("log enable lldb types")
+        self.expect('settings set symbols.swift-validate-typesystem false')
```

Comment:
Why do you need to turn off validation here? Perhaps it's better to special case it in TypeSystemSwiftTypeRef's SkipValidation function, or leave a comment explaining why.